google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -920,17 +920,7 @@ def default_acl
         #   policy = bucket.policy # API call
         #   policy_2 = bucket.policy # No API call
```

Comment:
I think we'd better keep this, since memoization is not typical in google-cloud-ruby.